### PR TITLE
Feat add new

### PR DESCRIPTION
**Pull request by weixuefeng**
## Checklist

- [ ] Create pull request as draft initially, unless its complete.
- [ ] Add tests to cover changes as needed.
- [ ] Update documentation as needed.
- [ ] If there is a related Issue, mention it in the description.

If you're adding a new blockchain

- [ ] I have read the [guidelines](https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria) for adding a new blockchain.

### CODE REVIEW

File: src/Ethereum/Address.cpp [hunk 1]
```diff
@@ -35,8 +35,8 @@ Address::Address(const Data& data) {
 }
 
 Address::Address(const PublicKey& publicKey) {
-    if (publicKey.type != TWPublicKeyTypeSECP256k1Extended) {
```
Comment (Hiwko7879) [inline src/Ethereum/Address.cpp:38]: Address::Address(const PublicKey& publicKey) {
    if (publicKey.type != TWPublicKeyTypeSECP256k1Extended) 
    }

**Comment by Kamranwew70** [comment]
Hi what is the case

در تاریخ دوشنبه ۱۵ اوت ۲۰۲۲،‏ ۲۲:۰۵ Hiwko7879 ***@***.***>
نوشت:

**Comment by hewigovens** [comment]
Thanks for your interests, have you checked our integration guide? https://developer.trustwallet.com/wallet-core/newblockchain#integration-criteria, we don't have plan to add it now

**Comment by berniceburgo** [comment]
Approved

**Pull request closed without merging**